### PR TITLE
tests: update failover tests

### PR DESCRIPTION
**Pull request by evgeniiz321**

### CODE REVIEW

File: pytest_tests/lib/helpers/complex_object_actions.py [hunk 1]
```diff
@@ -275,17 +275,15 @@ def get_nodes_with_object(
 
     nodes_list = []
     for node in nodes:
-        wallet = node.wallet.path
-        wallet_config = neofs_env.generate_cli_config(node.wallet)
```
Comment (roman-khimov) [inline pytest_tests/lib/helpers/complex_object_actions.py:279]: Unnecessary change.

File: pytest_tests/tests/failovers/test_failover_storage.py [hunk 2]
```diff
@@ -10,43 +10,33 @@
...
         stopped_nodes.remove(node)
 
     wait_all_storage_nodes_returned(neofs_env)
 
 
 @pytest.mark.failover
-@pytest.mark.skip(reason="These tests require multiple hosts to run")
 class TestFailoverStorage(NeofsEnvTestBase):
-    @allure.title("Lose and return storage node's host")
-    @pytest.mark.parametrize("hard_reboot", [True, False])
```
Comment (roman-khimov) [inline pytest_tests/tests/failovers/test_failover_storage.py:47]: You can still have hard/soft. IIRC you're TERMinating process normally, but you can also KILL it.

File: pytest_tests/lib/helpers/complex_object_actions.py (hunk 1 above)
Comment (evgeniiz321) [reply pytest_tests/lib/helpers/complex_object_actions.py:279]: Moved it to a separate commit, otherwise we are regenerating cli config every time this function is called.

File: pytest_tests/tests/failovers/test_failover_storage.py (hunk 2 above)
Comment (evgeniiz321) [reply pytest_tests/tests/failovers/test_failover_storage.py:47]: Added kill case as well.

**Review by roman-khimov — APPROVED** [review]
